GitUI/Plugin/FailToLoadPlugin.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Diagnostics;
+using System.Text.RegularExpressions;
+using GitUI.Properties;
+using GitUIPluginInterfaces;
+
+namespace GitUI
+{
+    public class FailToLoadPlugin : IGitPlugin
+    {
+        private readonly Exception _exception;
+        private static readonly Regex pluginNameRegex = new("\\\"GitExtensions.([^\"]+)\\\"");
+        private string _pluginName;
+        public FailToLoadPlugin(Exception exception)
+        {
+            _exception = exception?.Demystify();
+            try
+            {
+                Match match = pluginNameRegex.Match(exception.ToString());
+                if (match.Success)
+                {
+                    _pluginName = match.Value;
+                    Name = TranslatedStrings.FailedToLoadPlugin + ": " + _pluginName.Substring(0, Math.Min(50, match.Value.Length));
+                }
+                else
+                {
+                    Name = TranslatedStrings.FailedToLoadPlugin;
+                }
+            }
+            catch (Exception)
+            {
+                Name = TranslatedStrings.FailedToLoadPlugin;
+            }
+        }
+
+        public Guid Id { get; } = Guid.NewGuid();
+        public string? Name { get; init; }
+        public string? Description { get; } = TranslatedStrings.FailedToLoadPlugin;
+        public Image? Icon { get; } = Resources.bug;
+        public IGitPluginSettingsContainer? SettingsContainer { get; set; }
+        public bool HasSettings { get; } = false;
+
+        public bool Execute(GitUIEventArgs args)
+        {
+            DialogResult result = MessageBox.Show(string.Format(TranslatedStrings.FailedToLoadPluginPopupText, Truncate(_exception.ToString())), TranslatedStrings.FailedToLoadPlugin, MessageBoxButtons.OKCancel, MessageBoxIcon.Error);
```

Comment:
It was to limit the size of the stacktrace displayed but it is in fact useless because the stacktrace is always the same and less than the limit. Removed. Thanks.